packages/date-picker/src/DatePicker/DatePicker.spec.tsx

```diff
@@ -1415,39 +1420,24 @@ describe('packages/date-picker', () => {
               expect(yearInput).toHaveFocus();
             });
 
-            test('moves the cursor when the segment has a value', () => {
-              const { monthInput } = renderDatePicker({
+            test('focuses the previous segment when the segment has a value', () => {
+              const { yearInput, monthInput } = renderDatePicker({
                 value: testToday,
               });
               userEvent.click(monthInput);
               userEvent.keyboard('{arrowleft}');
-              expect(monthInput).toHaveFocus();
+              expect(yearInput).toHaveFocus();
             });
 
-            test('moves the cursor when the value starts with 0', () => {
-              const { monthInput } = renderDatePicker({});
+            test('focuses the previous segment when the value starts with 0', () => {
```

Comment:
don't think we need this test (or the following one) anymore. These are both covered by "...when the segment has a value"